lib/trento/hosts/host.ex

```diff
@@ -548,6 +548,12 @@ defmodule Trento.Hosts.Host do
 
   # Software Updates Discovery
 
+  def execute(
```

Comment:
We don't need this now, right?

Reply:
well does not harm and keeps away any possibility of weird things happening :smile: